mmf/utils/build.py

```diff
@@ -41,6 +44,116 @@ def build_model(config):
     return model
 
 
+def build_dataset(
```

Comment:
Can these methods go to `utils/dataset.py` ? `build.py` file is getting too large and have build_* methods ranging from trainer class to classifier layers. Might be good to move the build methods to respective utils class.

Reply:
Overall, I think keeping all of the build functions in a single file would be a better UX. Otherwise, user will need to know where exactly each function, even though may not be concerned about all utilities except build.py. I don't see an issue with file getting too large. (A lot of files in other projects are large as well, with IDE things should be easy to work with). I would like to keep all build functions together for now so that it is one stop-shop for anything you want to build.